src/app/ui/pressable/pressable.tsx

```diff
@@ -61,6 +61,7 @@ export const pressableStyles = css.raw({
     _before: {
       bg: 'ink.component-background-hover',
       borderColor: 'transparent',
+      transition: 'background 0.2s ease-in-out',
```

Comment:
This might get lost as it's being migrated to mono. Also, IMO some interactions shouldn't be animated. This animation, while it might look nice, delays the visual feedback to the user that the item is selected.

Should either remove, or make it _way_ faster, like 50ms